packages/overlay/src/overlay-trigger.ts

```diff
@@ -55,25 +61,42 @@ export class OverlayTrigger extends LitElement {
     public onOverlayOpen(event: Event, interaction: TriggerInteractions): void {
         const isClick = interaction === 'click';
         const overlayElement = isClick ? this.clickContent : this.hoverContent;
-        const delayAttribute = overlayElement
-            ? overlayElement.getAttribute('delay')
-            : null;
-        const delay = delayAttribute ? parseFloat(delayAttribute) : 0;
 
+        /* istanbul ignore if */
         if (!overlayElement) {
             return;
         }
+        if (!overlayRoot) {
+            overlayRoot = new OverlayRoot();
+        }
+
+        const delayAttribute = overlayElement.getAttribute('delay');
+        const delay = delayAttribute ? parseFloat(delayAttribute) : 0;
+
+        const queryThemeDetail: ThemeData = {
```

Comment:
Manage the nearest ancestor theming...allows us to treat that ancestor like the `overlay-root` (sort of). It's not 100% the same functionality, but I think it's more understandable to say "the closest `sp-theme` manages your styles" rather than "the closest `sp-theme` to the closest `overlay-root` manages your styles".